Makefile

```
@@ -1,5 +1,6 @@
 build_debug ?= false
 restart ?= all
+focus ?= .\*
```

```suggestion
focus ?= '.*'
```
nit: if you use quotes it should work as well